src/bp/ui-admin/src/Pages/Account/Register.tsx

```diff
@@ -0,0 +1,149 @@
+import { Button, FormGroup, InputGroup, Intent } from '@blueprintjs/core'
+import React, { Component, Fragment } from 'react'
+import { RouteComponentProps } from 'react-router'
+import { Redirect } from 'react-router-dom'
+import { Alert, Card, CardBody, CardText, CardTitle } from 'reactstrap'
+
+import logo from '../../media/nobg_white.png'
+import { setActiveWorkspace } from '../../Auth'
+
+interface Params {
+  strategy: string
+  workspace: string
+}
+
+type Props = {
+  auth: any
+} & RouteComponentProps<Params>
+
+interface State {
+  email: string
+  password: string
+  confirmPassword: string
+  registerUrl?: string
+  error?: string
+  isFirstUser: boolean
+}
+
+export default class Register extends Component<Props, State> {
+  state = {
+    email: '',
+    password: '',
+    confirmPassword: '',
+    registerUrl: undefined,
+    error: undefined,
+    isFirstUser: false
+  }
+
+  componentDidMount() {
+    // tslint:disable-next-line: no-floating-promises
+    this.loadAuthConfig()
+    setActiveWorkspace(this.props.match.params.workspace)
+  }
+
+  loadAuthConfig = async () => {
+    const strategyConfig = await this.props.auth.getStrategyConfig(this.props.match.params.strategy)
+    if (!strategyConfig) {
+      return this.setState({ error: 'Invalid strategy' })
+    }
+
+    this.setState({ ...strategyConfig })
+  }
+
+  register = async () => {
+    const { email, password, confirmPassword, registerUrl } = this.state
+
+    if (password !== confirmPassword) {
```

Comment:
should we validate policies in FE as well ?

Reply:
It's only basic validations, not sure we should focus on that at the moment, but it could be interesting